tests/utils/ops.py

```diff
@@ -7,3 +7,30 @@ def isclose(x1, x2, rtol=1e-5, atol=1e-8):
 
 def allclose(x1, x2, rtol=1e-5, atol=1e-8):
     return keras.ops.all(isclose(x1, x2, rtol, atol))
+
+
+def max_mean_discrepancy(x, y):
```

Comment:
In the future, let's put these in `bayesflow/metrics` directly. This time, I will move this and make it a bit more customizable.